libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/restricted/AuditRestrictedF2F.java

```diff
@@ -10,20 +10,17 @@
 @ExcludeFromGeneratedCoverageReport
 @Getter
 public class AuditRestrictedF2F implements AuditRestricted {
-    @JsonProperty("name")
     private final List<Name> name;
 
-    @JsonProperty("docExpiryDate")
     private String docExpiryDate;
 
-    public AuditRestrictedF2F(
-            @JsonProperty(value = "name", required = true) List<Name> name,
-            @JsonProperty(value = "docExpiryDate", required = true) String docExpiryDate) {
+    @JsonCreator
```

Comment:
For whatever reason, the new version doesn't like having to choose between the two constructors - so we have to help it out